README.rst

```diff
@@ -151,10 +151,36 @@ General
     logger = logging.getLogger("cli.logger")
     logger.info('hi')
 
-**Notes**: - All loggers set up and created before the initialization of the Cloud Foundry logging library will
-be left untouched. - When using Flask and Sanic with the logging library a before and
+**Notes**: All loggers set up and created before the initialization of the Cloud Foundry logging library will
+be left untouched. When using Flask and Sanic with the logging library a before and
 after request middleware is attached, and it will capture response times for each request.
 
+Setting and getting correlation ID
+""""""""""""""""""""""""""""""""""
+
+When using cf_logging in a web application you don't need to care about setting and getting correlation ID,
+because the logging frameworks supported will fetch it from HTTP headers and will set it to the log output for you.
+But if you are developing an application triggered via cli or messaging or by schedule, you might want to set the correlation ID manually,
```

Comment:
For non-web applications you have to set the correlation ID manually, as it is used for filtering.

Reply:
It is not mandatory to set correlation ID, only if app developer would like to. If available it might be used by application admin to filter the logs.

Reply:
For non-web applications you **could** to set the correlation ID manually, as it **could** be used for filtering.

Reply:
I think this is obvious

Reply:
sorry I was typing the message I think we should have in the readme.
"For non-web applications you could to set the correlation ID manually, as it could be used for filtering."
 instead of 
"But if you are developing an application triggered via cli or messaging or by schedule, you might want to set the correlation ID manually, so all log entries can be filtered lately based on this correlation_id."

sorter.